dlup/annotations.py

```diff
@@ -813,41 +949,13 @@ def as_geojson(self) -> GeoJsonDict:
         -------
         list of (str, GeoJsonDict)
         """
-        coordinates, size = self.bounding_box
-        region_size = (coordinates[0] + size[0], coordinates[1] + size[1])
-        all_annotations = self.read_region((0, 0), 1.0, region_size)
-
-        # We should group annotations that belong to the same class
-        grouped_annotations = []
-        previous_label = None
-        group = []
-        for annotation in all_annotations:
-            label = annotation.label
-            if not previous_label:
-                previous_label = label
-
-            if previous_label == label:
-                group.append(annotation)
-            else:
-                grouped_annotations.append(group)
-                group = [annotation]
-                previous_label = label
-        # After the loop, add the last group if it's not empty
-        if group:
-            grouped_annotations.append(group)
-
-        data: GeoJsonDict = {"type": "FeatureCollection", "features": [], "id": None}
-        for idx, annotation_list in enumerate(grouped_annotations):
-            label = annotation_list[0].label
-            if len(annotation_list) == 1:
-                json_dict = _geometry_to_geojson(annotation_list[0], label=label)
-            else:
-                if annotation_list[0].type in [AnnotationType.BOX, AnnotationType.POLYGON]:
-                    annotation = shapely.geometry.MultiPolygon(annotation_list)
-                else:
-                    annotation = shapely.geometry.MultiPoint(annotation_list)
-                json_dict = _geometry_to_geojson(annotation, label=label)
+        data: GeoJsonDict = {"type": "FeatureCollection", "properties": None, "features": [], "id": None}
```

Comment:
Thanks, that's a good addition. It's something which would be dlup-only so we need to handle this carefully and try to avoid too much custom properties. I would propose to have a dlup-xml format which can support this better.